fix(db): add back `FLOOR` so ttl returns integer instead of float (#9960)
Kong/kong: CHANGELOG.md

### Fixes

#### Core

> - Add back Postgres `FLOOR` function when calculating `ttl`, so it is always returned
>   as `int` instead of `float`.

Reviewer: ```suggestion
- Add back Postgres `FLOOR` function when calculating `ttl`, so the returned `ttl` is always a whole integer.
```
